db/seeds.rb

```diff
@@ -1,7 +1,66 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# admins
+FactoryBot.create(:admin, email: 'admin_freightage@mercadores.com.br')
+FactoryBot.create(:admin, email: 'admin_storage@mercadores.com.br')
+puts 'admins created'
+sleep(1)
+# companies
+first_shipping_co = FactoryBot.create(:shipping_co)
+second_shipping_co = FactoryBot.create(:shipping_co)
+puts 'shipping companies created'
+sleep(1)
+# employees
+FactoryBot.create(:employee, shipping_co: first_shipping_co)
+FactoryBot.create(:employee, shipping_co: second_shipping_co)
+puts 'employees created'
+sleep(1)
+# reach areas
+FactoryBot.create(:reach_area, shipping_co: first_shipping_co)
+FactoryBot.create(:reach_area, shipping_co: second_shipping_co)
+puts 'reach areas created'
+sleep(1)
+# price settings
+FactoryBot.create(:price_setting, shipping_co: first_shipping_co)
+FactoryBot.create(:price_setting, shipping_co: first_shipping_co)
+FactoryBot.create(:price_setting, shipping_co: second_shipping_co)
+FactoryBot.create(:price_setting, shipping_co: second_shipping_co)
```

Comment:
No price settings, vocês poderiam por favor definir cada um dos atributos, como fizeram em shipment time settings? Seria legal tbm ter alguns intervalos cadastrados de um par de atributos para o mesmo intervalo de outro par.

Por exemplo:
```suggestion
FactoryBot.create(:price_setting,
                    weight_min: 0, weight_max: 10,
                    volume_min: 0, volume_min: 3,
                    distance_value: 0.5,
                    shipping_co: second_shipping_co)
FactoryBot.create(:price_setting,
                    weight_min: 10, weight_max: 90,
                    volume_min: 0, volume_min: 3,
                    distance_value: 2.5,
                    shipping_co: second_shipping_co)
```

Reply:
feito